kpops/components/streams_bootstrap/__init__.py

```diff
@@ -29,3 +55,14 @@ class StreamsBootstrap(HelmApp, ABC):
         default=STREAMS_BOOTSTRAP_VERSION,
         description=describe_attr("version", __doc__),
     )
+
+    @pydantic.model_validator(mode="after")
+    def warning_for_latest_image_tag(self) -> Self:
+        if self.app.image_tag == "latest" and self.is_substituted():
+            log.warning(
```

Comment:
the warning is also raised when the image tag is explicitly set to 'latest' by the user, no?

Reply:
Yeah, that is true... 🤔 Do you know how you can differentiate between the default and user-set values in Pydantic?

Reply:
no, none that I am aware of. I do think the warning could make sense either way though. Would just need to he worded slightly differently